app/helpers/tel-format.js

```diff
@@ -1,8 +1,10 @@
 import { helper } from '@ember/component/helper';
 
 export default helper(function telFormat([tel]) {
```

Comment:
This is a helper function to be used in a template. You have to add type guarding to helper functions because it's difficult to foresee how the helper function will be used.
Make sure to add a guard if the type of the tel is not a string. Also mare sure there is an argument. Also you should always jump out of functions as early as possible.
```javascript
// First lines of function
if (!tel) throw new Error("Tel format helper requires one parameter");
if (typeof tel !=== 'string') throw new Error("Fel format helper expects a string as a parameter.");
```